code/controllers/subsystems/shuttles.dm

```diff
@@ -99,54 +98,51 @@ SUBSYSTEM_DEF(shuttles)
 		registered_shuttle_landmarks[shuttle_landmark_tag] = shuttle_landmark
 		last_landmark_registration_time = world.time
 
-		// TODO - Uncomment once overmap sectors are ported
-		//var/obj/effect/overmap/visitable/O = landmarks_still_needed[shuttle_landmark_tag]
-		//if(O) //These need to be added to sectors, which we handle.
-		//	try_add_landmark_tag(shuttle_landmark_tag, O)
-		//	landmarks_still_needed -= shuttle_landmark_tag
-		//else if(istype(shuttle_landmark, /obj/effect/shuttle_landmark/automatic)) //These find their sector automatically
-		//	O = map_sectors["[shuttle_landmark.z]"]
-		//	O ? O.add_landmark(shuttle_landmark, shuttle_landmark.shuttle_restricted) : (landmarks_awaiting_sector += shuttle_landmark)
+		var/obj/effect/overmap/visitable/O = landmarks_still_needed[shuttle_landmark_tag]
+		if(O) //These need to be added to sectors, which we handle.
+			try_add_landmark_tag(shuttle_landmark_tag, O)
+			landmarks_still_needed -= shuttle_landmark_tag
```

Comment:
To overmap shuttles (aka `visitable/ship/landable`s), yes, though there are reasons to have landmarks not visitable by overmap shuttles.

Visitable sectors (both ones that represent static physical locations, which I typically just call 'sectors', and ones that represent spaceships, which are `ship` and `ship/landable`) have a couple of lists on them: initial_generic_waypoints, and initial_restricted_waypoints. These are lists of the landmarks they should 'advertise' to things that arrive on top of them on the overmap, with the 'restricted' version only allowing a specific shuttle to specific landmarks.

So, 'overmap shuttles' will see all the landmarks in initial_generic_waypoints, and initial_restricted_waypoints (the latter, only ones it's allowed to see).

But, sometimes you might want a landmark in one of the z-levels that makes up an overmap sector for reasons other than overmap shuttles landing there. For example, if you have multishuttles, webshuttles, etc that use them, but don't want overmap ships landing at those spots, then you wouldn't add said landmarks to the initial_generic_waypoints or initial_restricted_waypoints lists, which would cause try_add_landmark_tag to 'fail' to add the landmark as 'advertised' by the sector. So overmap shuttles won't be able to land there, though the multi/web/ferry/whatever shuttles that use those landmarks will work fine.

Reply:
The "try" in `try_add_landmark_tag()` refers to the fact that it can only add a landmark to a sector (during sector initialization) if the landmark itself has already initialized.   If it "fails" it gets added to the `landmarks_still_needed` list so it will get hooked up later when the landmark itself initializes.
Note: There is also landmarks_awaiting_sector  list to handle the reverse situation (when landmark inits before it's sector)

So what this means is that the system is robust to sectors and landmarks initializing in any order.